### ISSUE TITLE
Wish to add htmlhint.config.(js|mjs) or htmlhint.config.json style configuration file name

### ISSUE THREAD

**Issue by hcpty** [htmlhint/HTMLHint#1468]
Since ESLint and Stylelint have htmlhint.config.(js|mjs) and htmlhint.config.json style configuration file name, we shall keep an uniform interface to a standard user experience.

**Comment by gian1200** [comment]
Looks similar/duplicate to https://github.com/htmlhint/HTMLHint/issues/126

**Comment by coliff** [comment]
Yep, I'll close this as a duplicate. Feel free to upvote for the issue with a thumbup on 126 !
